cmd/cli/telegram_utils.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/moira-alert/moira"
+	"github.com/moira-alert/moira/database/redis"
+	"github.com/moira-alert/moira/senders/telegram"
+)
+
+var telegramUsersKey = "moira-telegram-users:"
+
+func updateTelegramUsersRecords(logger moira.Logger, database moira.Database) error {
+	logger.Info().Msg("Start updateTelegramUsersRecords")
+
+	switch d := database.(type) {
+	case *redis.DbConnector:
+		pipe := d.Client().TxPipeline()
+		iter := d.Client().Scan(d.Context(), 0, telegramUsersKey+"*", 0).Iterator()
+		for iter.Next(d.Context()) {
+			key := iter.Val()
+			if key == "moira-telegram-users:moira-bot-host" {
```

Comment:
It is better to put such magic strings into constants, for example, `telegramLockName`

Reply:
Done https://github.com/moira-alert/moira/pull/1022/commits/42e4b55d4576b5bd3921bbbea3379782035f4457